repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/herp/HerpCloudfeedCADFdefaultTemplateTest.groovy

```diff
@@ -58,12 +58,14 @@ class HerpCloudfeedCADFdefaultTemplateTest extends ReposeValveTest {
     def "Events match filterout condition will not go to post filter log"() {
         setup: "declare messageChain to be of type MessageChain"
         def Map<String, String> actionmap = [
-                'GET'   : 'read/get',
-                'HEAD'  : 'read/head',
-                'POST'  : 'update/post',
-                'PUT'   : 'update/put',
-                'DELETE': 'update/delete',
-                'PATCH' : 'update/patch'
+                'GET'    : 'read/get',
+                'HEAD'   : 'read/head',
+                'OPTIONS': 'read/options',
+                'POST'   : 'update/post',
+                'PUT'    : 'update/put',
+                'DELETE' : 'update/delete',
+                'PATCH'  : 'update/patch',
+                'TRACE'  : 'unknown/trace'
```

Comment:
No need to change this, but just fyi, the quotations on the keys here are extraneous.  You only need them when there's a character in the key name (e.g. a dash).